internal/sbi/consumer/nrf_service.go

```diff
@@ -0,0 +1,137 @@
+/*
+ * NSSF Consumer
+ *
+ * Network Function Management
+ */
+
+package consumer
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"strings"
+	"time"
+
+	nssf_context "github.com/free5gc/nssf/internal/context"
+	"github.com/free5gc/nssf/internal/logger"
+	"github.com/free5gc/openapi"
+	"github.com/free5gc/openapi/Nnrf_NFManagement"
+	"github.com/free5gc/openapi/models"
+)
+
+type NrfService struct {
+	nrfNfMgmtClient *Nnrf_NFManagement.APIClient
```

Comment:
Add mutex 
```go
nfMngmntMu sync.RWMutex
```

Reply:
Why? No multiple connection at once?

Reply:
I know there would not be multiple connections at once. But I think it's better to have the same structure/logic as others.

Reply:
Or add a comment  to indicate that there would be no multiple-connection at once.

Reply:
Comment added